docs/getting-started-guides/docker.md

```diff
@@ -55,35 +55,7 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
-2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
How about keeping this around for folks who have their nodes in-correctly configured. This information could reside in a `Troubleshooting` section for example :)